tin-drink/src/components/content/drinkList/SingleDrink.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import './SingleDrink.css';
+
+function SingleDrink(props) {
+
+    console.log(props);
+    return(
+        <div id="single-drink" class="single-drink">
+            <div id="drink-photo" class="drink-photo">
+                <img src={props.drink.photoPath}/>
```

Comment:
img elements must have an alt prop, either with meaningful text, or an empty string for decorative images.